sdk/resourcemanager/Azure.ResourceManager/src/Resources/Generated/Extensions/ArmClientExtensions.cs

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using Azure.ResourceManager;
+
+namespace Azure.ResourceManager.Resources
```

Comment:
we shouldn't need this file when generating arm core.  I believe there is an if statement in autorest that adds this if the parent is a set number of parents, we should add an exception for when we are generating arm core

Same goes for ManagementGroup, ResourceGroup and Subscription

Reply:
Keep the extensions in this PR and will remove them when we can generate the pseudo-generated code in a follow-up PR where ManagementGroup, ResourceGroup and Subscription can be auto-generated (with necessary customization).